### ISSUE TITLE
Preview not working, vscode Linux

### ISSUE THREAD

**Issue by GabrielBRDeveloper** [AvaloniaUI/AvaloniaVSCode#58]
## About bug

The window preview not work, i did try **Create Assets** but not work.

When i open the **http://127.0.0.1:8000/** in browser it show the Preview normally.

### About Desktop

**Host system:** Linux Mint 21.1
**Vscode:** 1.83.1
**Avalonia:** 11.0.2
**Extension:** 0.0.20
**Dotnet Version:** 7.0.402

### Preview in vscode:

![image](https://github.com/AvaloniaUI/AvaloniaVSCode/assets/97042217/2e619df8-e99b-48dd-a881-dd4bd21807e0)

### Builded app:

![image](https://github.com/AvaloniaUI/AvaloniaVSCode/assets/97042217/a632ab6d-f1c5-4349-93e7-0047532b0722)

### Logs

```
2023-10-11 22:20:43.753 [info] Creating language service
2023-10-11 22:20:43.753 [info] Starting Avalonia Language Server...
2023-10-11 22:20:43.753 [info] [Trace - 22:20:43] Sending request 'initialize - (0)'.
2023-10-11 22:20:45.275 [info] [Trace - 22:20:45] Received response 'initialize - (0)' in 1527ms.
2023-10-11 22:20:45.276 [info] [Trace - 22:20:45] Sending notification 'initialized'.
2023-10-11 22:20:45.277 [info] [Trace - 22:20:45] Sending notification 'workspace/didChangeConfiguration'.
2023-10-11 22:20:45.361 [info] [Trace - 22:20:45] Received request 'client/registerCapability - (1)'.
2023-10-11 22:20:45.362 [info] [Trace - 22:20:45] Sending response 'client/registerCapability - (1)'. Processing request took 2ms
2023-10-11 22:20:45.362 [info] [Trace - 22:20:45] Sending notification 'textDocument/didOpen'.
2023-10-11 22:20:45.370 [info] [Trace - 22:20:45] Received request 'workspace/configuration - (2)'.
2023-10-11 22:20:45.371 [info] [Trace - 22:20:45] Sending response 'workspace/configuration - (2)'. Processing request took 0ms
2023-10-11 22:20:45.404 [info] [Trace - 22:20:45] Received request 'workspace/configuration - (3)'.
2023-10-11 22:20:45.404 [info] [Trace - 22:20:45] Sending response 'workspace/configuration - (3)'. Processing request took 1ms
2023-10-11 22:20:45.440 [info] [Trace - 22:20:45] Received request 'workspace/configuration - (4)'.
2023-10-11 22:20:45.440 [info] [Trace - 22:20:45] Sending response 'workspace/configuration - (4)'. Processing request took 1ms
2023-10-11 22:20:47.450 [info] [Trace - 22:20:47] Received notification 'window/logMessage'.
2023-10-11 22:20:47.450 [info] [Info  - 22:20:47] AvaloniaLanguageServer.Handlers.TextDocumentSyncHandler: ** DidOpenText: file:///home/gabriel/Projects/CSharp/HelloAvalonia/Hello%20Avalonia/MainWindow.axaml | Uri='file:///home/gabriel/Projects/CSharp/HelloAvalonia/Hello%20Avalonia/MainWindow.axaml'
2023-10-11 22:20:54.292 [info] Command avalonia.previewProcess, file:///home/gabriel/Projects/CSharp/HelloAvalonia/Hello%20Avalonia/MainWindow.axaml
2023-10-11 22:20:54.292 [info] PreviewServer.start /home/gabriel/Projects/CSharp/HelloAvalonia/Hello Avalonia/bin/Debug/net7.0/Hello Avalonia.dll
2023-10-11 22:20:54.301 [info] Previewer process started with args: exec,--runtimeconfig "/home/gabriel/Projects/CSharp/HelloAvalonia/Hello Avalonia/bin/Debug/net7.0/Hello Avalonia.runtimeconfig.json",--depsfile "/home/gabriel/Projects/CSharp/HelloAvalonia/Hello Avalonia/bin/Debug/net7.0/Hello Avalonia.deps.json" "/home/gabriel/.nuget/packages/avalonia/11.0.2/tools/netcoreapp2.0/designer/Avalonia.Designer.HostApp.dll",--method avalonia-remote,--transport tcp-bson://127.0.0.1:8001/,--method html,--html-url http://127.0.0.1:8000,"/home/gabriel/Projects/CSharp/HelloAvalonia/Hello Avalonia/bin/Debug/net7.0/Hello Avalonia.dll"
2023-10-11 22:20:54.301 [info] Preview server listening on port 8001
2023-10-11 22:20:54.303 [info] Show Preview to side: file:///home/gabriel/Projects/CSharp/HelloAvalonia/Hello%20Avalonia/MainWindow.axaml
2023-10-11 22:20:54.443 [info] Preview server connected on port 8001
2023-10-11 22:20:54.461 [info] Obtaining AppBuilder instance from Hello_Avalonia.Program

2023-10-11 22:20:54.478 [info] Initializing application in design mode

2023-10-11 22:20:54.527 [info] {"Uri":"http://127.0.0.1:8000/"}
2023-10-11 22:20:54.818 [info] Sending StartDesignerSessionMessage

2023-10-11 22:20:54.821 [info] {"SessionId":"c2b709f4-c47a-4bf9-a252-5f444c6ebb95"}
2023-10-11 22:20:54.821 [info] Start designer session message received.
2023-10-11 22:20:56.139 [info] {"Error":null,"Handle":null,"Exception":null}

```

**Comment by prashantvc** [comment]
Could you try restarting the VS Code? Do you see the same issue?

Can you share console output from `Help -> Toggle Developer Tools`, please? Keep it open before you open the previewer

**Comment by prashantvc** [comment]
Thanks a lot for sharing the logs
Looks like something specific to Mint, it's not loading the scripts in the previewer.

**Comment by prashantvc** [comment]
I am sorry you’re hitting these issues, I will get Mint on my machine to test it out.

**Comment by GabrielBRDeveloper** [comment]
Thank you very much, for help.

**Comment by heartacker** [comment]
> Other bug: auto complete, not work sometimes
> 
> **I did try**: Restart vscode, create assets, build project, click in _Build the project_
> 
> ![image](https://user-images.githubusercontent.com/97042217/274615244-7b64df3d-5935-414a-a390-d3bdad29a329.png)

same there

**Comment by prashantvc** [comment]
@heartacker are you on Linux too? What flavour of Linux are you on? Could you share the logs from `Avalonia Client` from the output?

**Comment by kev005** [comment]
> Other bug: auto complete, not work sometimes
> 
> **I did try**: Restart vscode, create assets, build project, click in _Build the project_
> 
> ![image](https://user-images.githubusercontent.com/97042217/274615244-7b64df3d-5935-414a-a390-d3bdad29a329.png)

Same in Fedora 38, but downgrading to 0.0.18 solves both issues (code completion and previewer).

**Comment by prashantvc** [comment]
> Same in Fedora 38, but downgrading to 0.0.18 solves both issues (code completion and previewer).

I am failing to reproduce it! However, what I noticed is that when you select `Build the project` for the second time it works. Could you confirm it's same for you?

**Comment by Mrcubix** [comment]
Seems to have been fixed in #64 the issue was apparently not platform specific

![image](https://github.com/AvaloniaUI/AvaloniaVSCode/assets/39861216/1f60d547-cc10-4cbb-bfb5-2cc78ead6f88)
![image](https://github.com/AvaloniaUI/AvaloniaVSCode/assets/39861216/0ef15e71-1a0f-472b-8d8f-df75a900ccbb)

**Comment by prashantvc** [comment]
Fixed in 0.0.25
